bundles/SyliusReportBundle/index.rst

```diff
@@ -0,0 +1,15 @@
+SyliusReportBundle
+===================
+
+SyliusReportBundle is the flexible reports management system for Symfony2 e-commerce applications.
```

Comment:
is the flexible => is **a** flexible

for Symfony2 e-commerce applications => for Symfony2 ~~**e-commerce**~~ applications